src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift

```diff
@@ -387,6 +388,77 @@ final class HealthCertificateViewModel {
 		.compactMap { $0 }
 	}
 
+	/// these strings here are on purpose not localized
+	///
+	private func updateRecoveryCertificateKeyValueCellViewModels(recoveryEntry: RecoveryEntry) {
+		let nameCellViewModel = HealthCertificateKeyValueCellViewModel(
+			key: "Name, Vorname / Name, First Name",
+			value: healthCertificate.name.fullName,
```

Comment:
have a look on #2949 , the name is displayed wrong here. Don't know how you want to resolve this (wait for other PR or just fix it here, too)